editor/src/components/navigator/navigator.tsx

```diff
@@ -17,6 +17,14 @@ import { useEditorState, useRefEditorState } from '../editor/store/store-hook'
 import { ElementContextMenu } from '../element-context-menu'
 import { createDragSelections } from '../../templates/editor-navigator'
 import { betterReactMemo } from 'uuiui-deps'
+import { FixedSizeList, ListChildComponentProps } from 'react-window'
+import { Size } from 'react-virtualized-auto-sizer'
+// There's some weirdness between the types and the results in the two module systems.
+// This is to effectively massage the result so that if it is loaded in the browser or in
+// node it should end up with the right thing.
+const AutoSizer = require('react-virtualized-auto-sizer')
```

Comment:
this is about the 4th time we encounter this problem. Last time I tried searching for a solution I didn't find anything, but that was a long time ago. It feels like we shouldn't be the only people who has this problem? surely other teams use `react-virtualized-auto-sizer` and also write tests, right? surely

Reply:
I guess if you make sure both sides are using the same working module system this becomes a non-problem.